src/js/components/Chart.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <div>
+    <svg id="chart" width="100%" height="500px">
+      <g>
+        <rect
```

Comment:
@stacyk @jgerigmeyer I'm wondering if it will be possible to draw these rectangles with the svg `path` element? Would the dynamic values that we're using work as coordinates? 

I ask because it seems like the `rx` and `ry` values that add the border radius to the `rect` element seem to apply the border-radius to _all_ corners. E.g. if `rx` is set to 3px, `ry` will also be 3px: https://medium.com/@dennismphil/one-side-rounded-rectangle-using-svg-fb31cf318d90

Still looking into how to only have some of the corners of this svg rounded.

Reply:
Yeah, I think you're right and that's how we did two rounded corners on Workflow Builder. I'm pretty sure @stacyk came up with that, so I'll let her weigh in with suggestions.

- https://github.com/oddbird/aunsight-workflow-builder/blob/master/src/components/graph/ProcessPorts.vue#L7-L11
- https://github.com/oddbird/aunsight-workflow-builder/blob/master/src/components/graph/ProcessType.vue#L8-L11
- 
![Screen Shot 2021-12-14 at 9 41 50 AM](https://user-images.githubusercontent.com/552316/146019770-c7973336-6f1d-4fd9-9c5f-b1c9d1dc8f21.jpg)
![Screen Shot 2021-12-14 at 9 41 57 AM](https://user-images.githubusercontent.com/552316/146019779-d603b815-e169-4045-bc0c-080aa2d118be.jpg)